### ISSUE TITLE
Allow passing arguments as test title

### ISSUE THREAD

**Issue by so1ve** [veritem/eslint-plugin-vitest#183]
e.g

```ts
const foo = "sth";
describe(foo, () => {});
```

I can send a pr to implement this

**Comment by veritem** [comment]
PR is always welcome!
